src/query/CMakeLists.txt

```
@@ -42,6 +42,9 @@ add_dependencies(mg-query generate_lcp_query)
 target_include_directories(mg-query PUBLIC ${CMAKE_SOURCE_DIR}/include)
 target_link_libraries(mg-query dl cppitertools)
 target_link_libraries(mg-query mg-integrations-kafka mg-storage-v2 mg-utils mg-kvstore mg-memory)
+if(NOT "${MG_PYTHON_PATH}" STREQUAL "")
+    set(Python3_ROOT_DIR "${MG_PYTHON_PATH}")
+endif()
 if("${MG_PYTHON_VERSION}" STREQUAL "")
     find_package(Python3 3.5 REQUIRED COMPONENTS Development)
```

Actually, what if path and version are both provided?
I would put 
```suggestion
if(NOT "${MG_PYTHON_PATH}" STREQUAL "")
    set(Python3_ROOT_DIR "${MG_PYTHON_PATH}")
else()
    if("${MG_PYTHON_VERSION}" STREQUAL "")
        find_package(Python3 3.5 REQUIRED COMPONENTS Development)
      ....
endif()
```

---

After rerunning everything again, I think the initial code is actually optimal because `find_package` still has to be called. In addition, there are multiple warnings during cmake run:
```
CMake Warning at src/CMakeLists.txt:47 (add_executable):
  Cannot generate a safe runtime search path for target memgraph because
  files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libcurl.so.4] in /usr/lib/x86_64-linux-gnu may be hidden by files in:
      miniconda3/envs/mageCuGraph/lib
    runtime library [libssl.so.1.1] in /usr/lib/x86_64-linux-gnu may be hidden by files in:
      miniconda3/envs/mageCuGraph/lib
    runtime library [libcrypto.so.1.1] in /usr/lib/x86_64-linux-gnu may be hidden by files in:
     miniconda3/envs/mageCuGraph/lib

  Some of these libraries may not be found correctly.
```
but as this https://stackoverflow.com/questions/22536567/cmake-cannot-resolve-runtime-directory-path thread explains (in particular this answer https://stackoverflow.com/a/69742501), we have to find a way to safely load these libraries (not a configure/build issue). I think we have to come up with e.g. implementation of `cmake/FindOpenSSL.cmake` but since https://github.com/Kitware/CMake/blob/master/Modules/FindOpenSSL.cmake is quite complex, I wouldn't go there for now as a part of this PR.